File: prototype/base/src/main/java/com/project/base/domain/AppUser.java
package com.project.base.domain;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;
import org.springframework.data.relational.core.mapping.Column;

@Data
@Table("app_user")
public class AppUser {

    @Id
    private Long id;

    @Column("username")
    private String username;

    @Column("password")
    private String password;

    @Column("roles")
    private String roles;

    @Column("email")
    private String email;

    @Column("phone_number")
    private String phoneNumber;
}

File: prototype/lietzsche/src/main/kotlin/com/stxtory/lietzsche/domain/AppUser.kt
package com.stxtory.lietzsche.domain

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table

@Table("users")
data class AppUser(
    @Id val id: Long? = null,
    val username: String,
    val password: String,           // BCrypt 해시 저장
    val roles: String = "ROLE_USER" // 콤마구분: "ROLE_USER,ROLE_ADMIN"
)